### PR TITLE
Update zope version

### PR DESCRIPTION
**Pull request by sallner**
This PR should allow a release which works with Zope 4 and Python 2.7. 
I did not manage to easily create a `buildout.cfg` to test this PR also with Zope 2.13.28, so I propose to leave the 1.0.4 the last version working with Zope 2.13.28 and Python 2.

There might be another major release directly afterwards, as Python 3 seems to need a lot of test adaptation, so it is not trivial to support both at the same time.

### CODE REVIEW

**Comment by sallner** [comment]
For some reason the travis integration is not enabled in this repository. I would be happy if it gets enabled.

**Comment by icemac** [comment]
@sallner I enabled TravisCI. To run it a new commit seems to be needed.

File: CHANGES.rst [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 
-1.1.0 - unreleased
+2.0 - unreleased
 ------------------
 
-* Update various imports to zope.formlib and require at least Zope 2.13.
+* Update various imports to zope.formlib.
 
-* Deal with forward compatibility with Zope 4.
+* Support Zope 4 only.
```
Comment (icemac) [inline CHANGES.rst]: This should be the in the first line as it is the most important change.

File: buildout.cfg [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 parts = test
-extends = https://raw.githubusercontent.com/zopefoundation/Zope/4.0a6/versions-prod.cfg
+extends = https://raw.githubusercontent.com/zopefoundation/Zope/master/versions-prod.cfg
 develop = .
 extensions = mr.developer
 sources = sources
 sources-dir = auto-checkout-sources
 auto-checkout =
-    Zope2
+    Zope
```
Comment (icemac) [inline buildout.cfg]: Is this actually currently needed or is the latest released version enough?

File: buildout.cfg [hunk 3]
```diff
@@ -15,9 +15,9 @@ eggs =
 defaults = ['-c', '--module', 'five.formlib']
 
 [sources]
-Zope2 = git https://github.com/zopefoundation/Zope
+Zope = git https://github.com/zopefoundation/Zope
 
 [versions]
-Zope2 =
+Zope =
 mr.developer = 1.38
```
Comment (icemac) [inline buildout.cfg]: I think this pin can get deleted.

File: buildout.cfg [hunk 4]
```diff
@@ -15,9 +15,9 @@ eggs =
 defaults = ['-c', '--module', 'five.formlib']
 
 [sources]
-Zope2 = git https://github.com/zopefoundation/Zope
+Zope = git https://github.com/zopefoundation/Zope
 
 [versions]
-Zope2 =
+Zope =
```
Comment (icemac) [inline buildout.cfg]: This should be done by `mr.developer` if it is actually needed.

File: buildout.cfg [hunk 5]
```diff
@@ -15,9 +15,9 @@ eggs =
...
 mr.developer = 1.38
-zope.formlib = 4.4
+zope.formlib = >= 4.4
```
Comment (icemac) [inline buildout.cfg]: Maybe this should be moved to `setup.py` as install constraint.

Comment (sallner) [reply buildout.cfg]: That was already present. I deleted the whole version section.

File: buildout.cfg (hunk 4 above)
Comment (sallner) [reply buildout.cfg]: Deleted.

File: buildout.cfg (hunk 3 above)
Comment (sallner) [reply buildout.cfg]: Deleted.

File: CHANGES.rst (hunk 1 above)
Comment (sallner) [reply CHANGES.rst]: Changed.

File: buildout.cfg (hunk 2 above)
Comment (sallner) [reply buildout.cfg]: I removed the checkout.

**Pull request merged**